pkg/rlptools/wasm_utils.go

```diff
@@ -26,6 +26,23 @@ type GatewayAction struct {
 	Rules []kuadrantv1beta1.Rule `json:"rules,omitempty"`
 }
 
+func DefaultGatewayConfiguration(key client.ObjectKey) []kuadrantv1beta1.Configuration {
+	return []kuadrantv1beta1.Configuration{
+		{
+			Actions: []kuadrantv1beta1.ActionSpecifier{
+				{
+					GenericKey: &kuadrantv1beta1.GenericKeySpec{
+						DescriptorValue: key.String(),
+						// using default value as specified in Envoy spec
+						// https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#config-route-v3-ratelimit-action-generickey
+						DescriptorKey: &[]string{"generic_key"}[0],
```

Comment:
What about the following instead?

```suggestion
						DescriptorKey: &[]string{"ratelimitpolicy"}[0],
```

Reply:
does not make any diff and I was using the envoys RLS default value. It could be anything, it should not conflict as it is namespaced per gateway *and* domain

Reply:
> does not make any diff 

At least it means something for one who reads yielded config. "generic_key" means nothing IMO.

Reply:
done